packages/airnode-validator/src/config/config.test.ts

```diff
@@ -124,6 +124,16 @@ describe('nodeSettingsSchema', () => {
         region: 'region',
         disableConcurrencyReservations: false,
       },
+      httpGateway: {
```

Comment:
👍 I completely missed that I was not using any keys when implementing the unit test .